selfdrive/sensord/tests/test_sensord.py

```diff
@@ -158,6 +158,11 @@ def test_lsm6ds3_100Hz(self):
     data_list.sort()
     tdiffs = np.diff(data_list)
 
+    # with the skip of first 500ms measurements it happens that one value
+    # before the skip is received, this high diffs are filtered as they ruin
+    # the standard dev calculation
+    tdiffs = list(filter(lambda d: d <= 500*10**6, tdiffs))
```

Comment:
this shouldn't be necessary anymore, I added a 2s sleep in #25818

Reply:
retested, yes you are right